osu.ElasticIndexer/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build-env
+WORKDIR /app
+
+COPY *.csproj ./
+
+RUN dotnet restore
+
+COPY . ./
+RUN dotnet publish -c Release -o out
+
+FROM mcr.microsoft.com/dotnet/runtime:6.0
+WORKDIR /app
+COPY --from=build-env /app/out .
+CMD ["dotnet", "osu.ElasticIndexer.dll"]
```

Comment:
Any reason for using `CMD` and not `ENTRYPOINT`? It makes working with the project a bit weird.

Reply:
`CMD` was easier to override when doing something like `docker compose run indexer bash` to check what was going on int he container

Reply:
Fine if I change this across? You should be able to use `docker exec -it <container> bash` for that.